synapse/lib/modelrev.py

```diff
@@ -488,6 +490,76 @@ async def revModel20220706(self, layers):
         await self._propToForm(layers, 'it:av:sig:name', 'it:av:signame')
         await self._propToForm(layers, 'it:av:filehit:sig:name', 'it:av:signame')
 
+    async def revModel20220803(self, layers):
+
+        meta = {'time': s_common.now(), 'user': self.core.auth.rootuser.iden}
+
+        valid = regex.compile(r'^[0-9a-f]{40}$')
+        repl = regex.compile(r'[\s:]')
+
+        nodeedits = []
+        for layr in layers:
+
+            async def save():
+                await layr.storNodeEdits(nodeedits, meta)
+                nodeedits.clear()
+
+            formname = 'crypto:x509:cert'
+            prop = self.core.model.prop('crypto:x509:cert:serial')
+
+            async def movetodata(buid, valu):
+                nodeedits.append(
+                    (buid, formname, (
+                        (s_layer.EDIT_PROP_DEL, (prop.name, valu, prop.type.stortype), ()),
+                        (s_layer.EDIT_NODEDATA_SET, ('serial:0:2:10', valu, None), ()),
```

Comment:
In the event of two cert serial properties on a node that have to be moved, the EDIT_NODEDATA_SET can collide. Can we key something like `migration:0_2_10:{prop.name}` ?

Reply:
That does make it a bit difficult to know if there are multiple migration keys to check.
Maybe we roll up the errors into a single edit_nodedata_set ?

Reply:
Yeah, there's only one prop migrated here but in general merging them into a single `migration:{version}` key might make it easier to deal with